Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM bitwalker/alpine-erlang
```

Comment:
@archSeer the release contains *everything you need to run the application. However, I personally also included **ncurses** (for the terminal) and **openssl** (for DB connection)